src/pykiso/cli.py

```diff
@@ -90,12 +90,41 @@ def check_file_extension(
     return paths
 
 
+class CommandWithOptionalFlagValues(click.Command):
+    def parse_args(self, ctx, args):
+        """Translate any flag `--junit=value` as flag `--junit` with changed flag_value=value"""
+        # filter flags
+        flags = [
+            flag
+            for flag in self.params
+            if isinstance(flag, click.Option)
+            and flag.is_flag
+            and not isinstance(flag.flag_value, bool)
+        ]
+        junit_prefixes = {
+            flag_str: flag
+            for flag in flags
+            for flag_str in flag.opts
+            if flag_str.startswith("--junit")
+        }
+
+        for index, arg in enumerate(args):
+            arg = arg.split("=")
+            if len(arg) < 2:
```

Comment:
```suggestion
            if len(arg) != 2:
```
Even stricter, even better :D